applications/luci-app-ddns/luasrc/view/ddns/system_status.htm

```diff
@@ -95,56 +89,62 @@
 				network = '<em><%:Config error%></em>';
 			else
 				network = data[j].iface;
-
-			tbl.appendChild(E('<div class="tr cbi-section-table-row cbi-rowstyle-%d">'.format(((j + i) % 2) + 1), [
-					E('<div class="td">', [ E('<strong>', configuration) ]),
-					E('<div class="td">', next_update),
-					E('<div class="td">', lookup),
-					E('<div class="td">', registered_ip),
-					E('<div class="td">', network)
-					]));
-		}
-
-		if (tbl.firstElementChild === tbl.lastElementChild || (data[0].enabled == 0 && tbl.childNodes.length == 2) ) {
-			var br = '<br />';
-			var msg = '<%:There is no service configured.%>';
-			if (tbl.firstElementChild !== tbl.lastElementChild)
-				br = '';
-			msg = br + msg;
-			tbl.appendChild(E('<div class="tr cbi-section-table-row">', [
-					E('<div class="td">', [ E('<em>', msg) ])
-					]));
+			
+			rows.push([
+					'<strong>' + configuration + '</strong>',
+					next_update,
+					lookup,
+					registered_ip,
+					network
+					]);
+					
+			if (rows.length > 1) {
+				var legendtb = document.getElementById('ddns_status_table_legend');
+				legendtb.setAttribute("style", "display:table-row");
```

Comment:
Why is this needed?

Reply:
I notice that the legend is displayed anyway even if there are no data to show...  it should be hided since is useless...